rust/private/BUILD.bazel

```diff
@@ -1,6 +1,7 @@
 load("@bazel_skylib//:bzl_library.bzl", "bzl_library")
 load("//rust/private:rust_analyzer.bzl", "rust_analyzer_detect_sysroot")
 load("//rust/private:stamp.bzl", "stamp_build_setting")
+load(":utils_test.bzl", "utils_test_suite")
```

Comment:
Could you please move this file to `//test/unit`? That's where we typically leave all our tests. There is a potential future where we will be creating `rules_rust` releases and those won't have tests bundled. If we separate all tests to a side it will be easie achieve. If you're fighting with buildifier visibility, it is fine to disable buildifier visibility check for the test (`# buildifier: disable=bzl-visibility`)

Reply:
Done.